src/language-js/print/module.js

```diff
@@ -94,7 +94,20 @@ function printModuleSpecifiers(path, options, print) {
   return concat(parts);
 }
 
+function printImportAssertions(path, options, print) {
+  const node = path.getNode();
+  if (Array.isArray(node.assertions) && node.assertions.length !== 0) {
+    return concat([
+      " assert { ",
+      join(", ", path.map(print, "assertions")),
+      " }",
+    ]);
+  }
+  return "";
+}
```

Comment:
We can't distinguish these two, right?

**Prettier pr-9597**
[Playground link](https://deploy-preview-9597--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACAKjwEMBnPAMwgkuwgFs8AdEKiAOgCtToWmp0WXIRLkARsWy0GzEBOxceUFqNJxhwPHgC+IADQgIGGAEtopZKEl0A7gAVJCCymIAbG8QCeFg2OzEwAGs4GABlDACTKABzZBhsAFc4Azh6MTgAEwzMgBliGITiaLgAMRx6YhhTGOQQYgSYCH0QAAsYelcAdRaTeFIIsDhQp16TADdez1qwUh8QKLVcO39oiuQKNzUDbjQAIX8gkNDiejgcqLh1zeSQHdCo6Nc4AEUEiHgr1y2QCOxF2ok6VczQw2CiME6JgyMBayAAHAAGAygiBqTr+DC1UFwRZjS4GACOb3gyyMzjqpAAtAJMplmtg4ESTAzlkU1kgNl8bmp6CY4oluQ8nq93pcOdcDDBiGJIdDYUgAEyS-wmVwPADCDHZqFIAFZmgk1AAVaXOTnfMZJACSUGysFCYDBxgAgrbQjBPE9PmptNogA)
```sh
--parser babel
```

**Input:**
```jsx
export * as foo from "foo.json"
export * as bar from "bar.json" assert {  }
```

**Output:**
```jsx
export * as foo from "foo.json";
export * as bar from "bar.json";

```

Reply:
Tests added, for future improve.